> please fix the following warnings:
> 
> ```
> [build] ../src/client/network/mqtt/mqtt_mosquitto.c:145:1: warning: non-void function does not return a value [-Wreturn-type]
> [build] }
> [build] ^
> ```
> 
> ```
> [build] ../src/client/api/events/node_event.c:65:37: warning: implicit conversion from enumeration type 'mqtt_event_id_t' to different enumeration type 'event_client_event_id_t' [-Wenum-conversion]
> [build]       node_event->event_id = event->event_id;
> [build]                            ~ ~~~~~~~^~~~~~~~
> ```

Done.

--- a/src/client/api/events/node_event.c
+++ b/src/client/api/events/node_event.c
@@ -5,8 +5,8 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#include "client/api/events/node_event.h"
 #include "client/network/mqtt/mqtt.h"
-#include "node_event.h"
 
 /**
  * @brief Event client session

